pep-0206.txt

```diff
@@ -5,112 +5,124 @@ Last-Modified: $Date$
+Currently this document doesn't suggest *removing* modules from
+the standard library that are superseded by a third-party module.
+That's difficult to do because it entails many backward-compatibility
+problems, so it's not worth bothering with now.
+
+Please suggest additional domains of interest.
 
 
 Domain: Web tasks
+=================
 
-    XML parsing: ElementTree + SAX.
+XML parsing: ElementTree + SAX.
 
-    URL retrieval: libcurl? other possibilities?
+URL retrieval: libcurl? other possibilities?
 
-    HTML parsing: mxTidy? HTMLParser?
+HTML parsing: mxTidy? HTMLParser?
 
-    Async network I/O: Twisted
+Async network I/O: Twisted
 
-    RDF parser: ???
+RDF parser: ???
 
-    HTTP serving: ???
+HTTP serving: ???
 
-    HTTP cookie processing: ???
+HTTP cookie processing: ???
 
-    Web framework: A WSGI gateway, perhaps?  Paste?
+Web framework: A WSGI gateway, perhaps?  Paste?
 
-    Graphics: PIL, Chaco.
+Graphics: PIL, Chaco.
 
 
 Domain: Scientific Programming
+==============================
 
-    Numeric: Numeric, SciPy
+Numeric: Numeric, SciPy
 
-    Graphics: PIL, Chaco.
+Graphics: PIL, Chaco.
 
 
 Domain: Application Development
+===============================
 
-    GUI toolkit: ???
+GUI toolkit: ???
 
-    Graphics: Reportlab for PDF generation.
+Graphics: Reportlab for PDF generation.
 
 
 Domain: Education
+=================
 
-    Graphics: PyGame
+Graphics: PyGame
 
 
 Software covered by the GNU General Public License
+==================================================
+
+Some of these third-party modules are covered by the GNU General
+Public License and the GNU Lesser General Public License.
+Providing a script to download and install such packages, or even
+assembling all these packages into a single tarball or CD-ROM,
+shouldn't cause any difficulties with the GPL, under the "mere
+aggregation" clause of the license.
 
-    Some of these third-party modules are covered by the GNU General
-    Public License and the GNU Lesser General Public License.
-    Providing a script to download and install such packages, or even
-    assembling all these packages into a single tarball or CD-ROM,
-    shouldn't cause any difficulties with the GPL, under the "mere
-    aggregation" clause of the license.
-   
 
 Open Issues
+===========
 
-    What other application domains are important?
+What other application domains are important?
 
-    Should this just be a set of Ubuntu or Debian packages?  Compiling
-    things such as PyGame can be very complicated and may be too
-    difficult to automate.
+Should this just be a set of Ubuntu or Debian packages?  Compiling
+things such as PyGame can be very complicated and may be too
+difficult to automate.
 
 
 Acknowledgements
-        
-    The PEP is based on an earlier draft PEP by Moshe Zadka, titled
-    "2.0 Batteries Included."
+================
+
+The PEP is based on an earlier draft PEP by Moshe Zadka, titled
+"2.0 Batteries Included."
 
```

Comment:
Thanks @gvanrossum :)
I won't make changes wrt copyright notice without @akuchling 's input.
If anything, perhaps it should go in a separate PR anyway?